analytics/terraform/spark-eks-ipv6/eks.tf

```
@@ -0,0 +1,110 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "~> 19.15"
+
+  cluster_name                   = local.name
+  cluster_version                = var.eks_cluster_version
+  cluster_endpoint_public_access = true
+
+  # IPV6
+  cluster_ip_family          = "ipv6"
+  create_cni_ipv6_iam_policy = true
```

```suggestion
  create_cni_ipv6_iam_policy = false
```

the creation of this solution failed in my account as i already had the `AmazonEKS_CNI_IPv6_Policy` created. 
https://github.com/terraform-aws-modules/terraform-aws-eks/issues/2131 

We should probably call out that this policy needs to be created as a pre-req as described here https://docs.aws.amazon.com/eks/latest/userguide/cni-iam-role.html#cni-iam-role-create-ipv6-policy

---

Removed completely. Default is false anyway. Added comment in prereq to create the policy. Good call!